tests/grid-constructor/container.js

```diff
@@ -20,10 +20,6 @@
       new Muuri(document.documentElement);
     }, 'Should throw an error when documentElement is set as container');
 
-    assert.throws(function () {
```

Comment:
We don't need to remove this test if we keep the original idea in the Grid.js check intact.

Reply:
And this test will be reverted